modules/markdown/easy-markdown.php

```diff
@@ -72,7 +72,7 @@ public function load() {
 		if ( current_theme_supports( 'o2' ) || class_exists( 'P2' ) ) {
 			$this->add_o2_helpers();
 		}
-		jetpack_register_block( 'markdown' );
+		register_block_type( 'jetpack/markdown' );
```

Comment:
shouldn't we check if `register_block_type` exists here, before to use it?

Reply:
I'm kind of confident that we're only calling these from places that are wrapped by such a check somewhere up the call stack since they'd otherwise have probably come crashing down hard during unit tests or manual smoke testing (since a lot of other things did) :sweat_smile: but I'll give them another round of vetting.

Reply:
Added the check per 3f93ab58b